pdc/apps/contact/migrations/0006_auto_20151127_0837.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contact', '0005_auto_20151027_0725'),
+    ]
+
```

Comment:
Good point. Actually this file is generated automatically by command "python manage.py makemigrations/migrate". Django framework will build a migration dependency graph using both the disk migration files and database. so i am assuming the framework will take care the dependency issue and it will not fail the migration.